app/helpers/client_helper.rb

```diff
@@ -1,33 +1,27 @@
 # Contains functions which make client selection/branching easier
 module ClientHelper
-  def client_partial(client_name, path, args={})
-    client_name ||= "default"
-    to_check = client_name + "/" + path
+  def client_partial(client_slug, path, args={})
+    client_slug ||= "default"
+    to_check = client_slug + "/" + path
     default_check = "default/" + path
+
     if lookup_context.template_exists?(to_check, [], true)
       args[:partial] = to_check
       render(args)
     elsif lookup_context.template_exists?(default_check, [], true)
       args[:partial] = default_check
       render(args)
-    else
-      ""
```

Comment:
yeah it could. right now, no tests are breaking so I am assuming it is not breaking anything at the moment. 

I think the benefit if returning nothing if there is not a partial is that things *will* break when there is not a `client_partial` for something. my worry with the method as it was is that it would work whether a partial existed or not, but we always had partials for the cases where we were using this method.

If there ends up being a case where we want to render nothing, I would put in an empty template at the expected path. does that make sense?